lib/ansible/modules/cloud/smartos/vmadm.py

```diff
@@ -33,7 +33,7 @@
       - Whether or not a VM is booted when the system is rebooted.
   brand:
     required: true
-    choices: [ joyent, joyent-minimal, kvm, lx ]
+    choices: [ joyent, joyent-minimal, lx, kvm, bhyve ]
```

Comment:
add a note in description stating that 'bhyve' is supported from 2.10 on-wards.

Reply:
done